src/Umbraco.Core/IO/MediaFileSystem.cs

```diff
@@ -68,5 +76,16 @@ public void DeleteThumbnails(string path)
 			GetThumbnails(path)
 				.ForEach(DeleteFile);
 		}
+
+	    public void CopyThumbnails(string sourcePath, string targetPath)

```

Comment:
No, not part of IFileSystem - I have not modified IFileSystem. At the moment, we do explicitely use MediaFileSystem in various places, due to thumbnails, so I thought I'd add that method here too. The day we fully stop mananing thumbnails we can remove it?